docker-compose.prod.yml

```diff
@@ -16,3 +19,40 @@ services:
 
   api:
     restart: always
+
+  traefik:
+      container_name: traefik
+      image: traefik:v2.0
+      command:
+          - "--api.dashboard=true"
+          - "--providers.docker=true"
+          - "--entryPoints.web.address=:80"
+          - "--entryPoints.websecure.address=:443"
+          - "--certificatesResolvers.le.acme.email=${TRAEFIK_EMAIL}"
+          - "--certificatesResolvers.le.acme.storage=acme.json"
+          - "--certificatesResolvers.le.acme.tlsChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge=true"
+          - "--certificatesResolvers.le.acme.httpChallenge.entryPoint=web"
+          - "--log.level=INFO"
+      restart: always
+      ports:
+          - 80:80
+          - 443:443
+          - 8080:8080
```

Comment:
What is this port for?

Reply:
Traefik dashboard, it's useful while settings things up, we can remove it after